packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -88,8 +87,7 @@ export const useGetHistoryChats = (): UseGetHistoryChatsResult => {
 		const archived: Conversations = [];
 
 		mergedAndSortedConversations.forEach( ( conversation ) => {
-			const createdAt = getConversationCreatedAt( conversation );
-
+			const createdAt = Number( conversation.messages?.[ 0 ]?.received ) * 1000 || 0;
```

Comment:
I have a similar remark than my previous comment about renaming that variable to `lastMessageReceivedAt`.